src/router/index.ts

```diff
@@ -60,6 +61,12 @@ const routes: Array<RouteRecordRaw> = [
     component: Orders,
     beforeEnter: authGuard
   },
+  {
+    path: '/job-details',
+    name: 'JobDetail',
+    component: JobDetail,
```

Comment:
make "Detail" plural
```suggestion
    component: JobDetails,
```

Reply:
Ok sir I have renamed it.